azure/templates/post-deploy.yml

```diff
@@ -0,0 +1,50 @@
+parameters:
+  - name: aws_dev
+    default: aws --profile=apim-dev
+  - name: is_ptl
+    default: true
+
+steps:
+  - ${{ if parameters.is_ptl }}:
+    - template: "azure/components/aws-assume-role.yml@common"
+      parameters:
+          role: "auto-ops"
+          profile: "apm_ptl"
+
+  - bash: |
+      make install-python
+
+    workingDirectory: $(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)
+    displayName: Setup pytests
+    condition: always()
+
+  - bash: |
+      set -e
+      export RELEASE_RELEASEID=$(Build.BuildId)
+      export SOURCE_COMMIT_ID=$(Build.SourceVersion)
+      export APIGEE_ENVIRONMENT="$(ENVIRONMENT)"
+      export SERVICE_BASE_PATH="$(SERVICE_BASE_PATH)"
+      export APIGEE_ACCESS_TOKEN="$(secret.AccessToken)"
+      export PROXY_NAME="$(FULLY_QUALIFIED_SERVICE_NAME)"
+      export STATUS_ENDPOINT_API_KEY="$(status-endpoint-api-key)"
+
+      if [[ $APIGEE_ENVIRONMENT =~ .*-*sandbox ]]
```

Comment:
This if condition can be simplified by setting only marker. Also please remove the int one. We can add it later when it's needed.